README.md

```diff
@@ -101,7 +101,19 @@ $ nodemon bin/www
 
 ## Static Code Analysis Tool 
 
-- JSLint
+- ESLint
+> Install ESLint dependencies
+
+```shell
+$ npm init @eslint/config
```

Comment:
I think this creates the config file? We should already have one in our repo so this *might* overwrite it.

i.e. remove

Though, if I'm wrong, can you add config flags so the command line skips the interactive install parts? If you run this as-is it asks a couple questions on how it should be set up. We want everyone with the same setup.

Reply:
Removing the config command fixes the issue for me. How's it look on your system?

Reply:
It works for me, though I already set everything up for the last PR. It would be great to have someone with a fresh box to try it out.

Likely one of the 3 from #47 ?

Reply:
Ill ping them in Discord